phpunit.xml.dist

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- https://phpunit.de/manual/current/en/appendixes.configuration.html -->
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.2/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="vendor/autoload.php"
+>
+    <php>
+        <ini name="error_reporting" value="-1"/>
+        <server name="KERNEL_CLASS" value="PhpList\PhpList4\Core\ApplicationKernel"/>
```

Comment:
This now needs to be `PhpList\Core\ApplicationKernel`, not `PhpList\PhpList4\Core\ApplicationKernel` anymore, now that #275 is merged.

Reply:
Are you sure that doesn't cause any errors? I still can't find any class named `PhpList\Core\ApplicationKernel`

https://github.com/phpList/core/blob/master/src/Core/ApplicationKernel.php

Reply:
Oh, my bad. We haven't changed the PHP namespace yet. So the class name in the file still is correct.